### PR TITLE
Use chruby ruby versions in docs

### PR DESCRIPTION
**Pull request by DanToml**

### CODE REVIEW

File: software_versions.rb [hunk 1]
```diff
@@ -79,7 +84,7 @@ def ruby()
      version: version}
   end
   {
-    version: output('ruby -v'),
+    version: chruby_installed_rubies.join(", "),
```
Comment (marcomorain) [inline software_versions.rb]: I'd make this an array.

File: software_versions.rb [hunk 2]
```diff
@@ -71,6 +71,11 @@ def homebrew()
   end
 end
 
+def chruby_installed_rubies()
+  require 'pathname'
+  Dir.glob('/Users/distiller/.rubies/ruby-*').map {|p| Pathname.new(p).basename.to_s.sub('ruby-', '') }
```
Comment (marcomorain) [inline software_versions.rb]: Can you use `$HOME` here? I run this script locally during dev, and I'm not called `distiller` 😹

File: software_versions.rb (hunk 1 above)
Comment (DanToml) [reply software_versions.rb]: I was going to do that, but then I'd need nested tables in the output :grimacing:

File: software_versions.rb (hunk 2 above)
Comment (DanToml) [reply software_versions.rb]: lol sure

**Pull request merged**